lib/collection/src/collection.rs

```diff
@@ -366,7 +366,7 @@ impl Collection {
                         debug_assert!(!was_not_transferred);
                         false // Shard if already in transferring state
                     }
-                    Shard::ReplicaSet(_) => todo!(),
+                    Shard::ReplicaSet(_) => false, // Replica set can not be transfered
```

Comment:
In my ongoing WIP PR I am using this function for the replica set syncing.

The shard can be "transferred" if 

```rust
 Shard::ReplicaSet(replica_set) => sync && replica_set.local.is_some(),
 ```

`sync` being a new flag that I propagate from the `ShardTransfer` consensus message.

WDYT about this appproach?

Reply:
Yes, can be like that, I don't mind this approach

Reply:
Do you want me to keep a `todo` in this PR?